### DISCUSSION TITLE
Message popup keeps from seeing typed text

### DISCUSSION THREAD

**Discussion by bobromeo** [dbeaver/dbeaver#16320] [Q&A]
I am using dBeaver EE 22.0.0 on WinServer 2019.
When a query ends i get a win-like popup with a message, i think it lasts about 3 seconds.
This popup keeps me from seeing what i am typing ?!?
I can't find anything in the preferences on this behavior, which is extra annoying when i cancel a query :-)

Can anyone relate to this ?

**Comment by ShadelessFox** [comment]
Hello @bobromeo,

Unfortunately, from this brief description it's unclear what it can be.

Can you please record a video showing the issue?

**Comment by bobromeo** [comment]
It is hard to replicate, but now i think it is when there's also an error which says Stop/Skip etc.

**Reply by bobromeo** [reply]
I think my prior remark is true, but just now i see when a query is finished (correctly) in an other tab, i am not seeing typed text in my current tab
